### PR TITLE
Setup OwnerApp #2

### PR DESCRIPTION
**Pull request by yvanhenang**

### CODE REVIEW

**Comment by NkwaTambe** [comment]
Hello @yvanhenang  please add a description to your issue

File: tsconfig.node.tsbuildinfo [hunk 1]
```diff
@@ -0,0 +1 @@
+{"root":["./vite.config.ts"],"version":"5.6.2"}
```
Comment (NkwaTambe) [inline tsconfig.node.tsbuildinfo]: We don't need these too...

it can always be generated locally for test when we need it

**Review by NkwaTambe — CHANGES_REQUESTED** [review]
Hello @yvanhenang , I noticed that your project structure is missing a **manifest file** and a **service worker file**. These are important, and I believe they are typically generated automatically during the React Vite project build process.

You should be able to find the manifest file referenced in the **vite.config.ts**, and the service worker file is generally located in the **src** folder **(e.g., sw.jsx)**. Could you look into adding these?

please use these to understand what i am saying, you can build a sample project to see :

Since we're using react and typescript, use the following to create your project
`npm create @vite-pwa/pwa@latest . --template react-ts`
and follow the prompts. Then 
`npm install` 
to add the pwa plugin and other dependencies
then head to `vite-config.ts` and locate the line with with the `devOptions.` Now change the value of the `enabled` property from `false` to `true.`
Next,
run `npm run dev` to confirm it is downloadable as a PWA.
After confirmation,
Clean up the project by removing all the components and stylings that came with the initialization, leaving a blank page on your browser.
Then commit your changes and push them to the remote repo.

**Comment by NkwaTambe** [comment]
Hello @yvanhenang  please feel free to ask for help from any of us, thanks

File: tsconfig.node.tsbuildinfo (hunk 1 above)
Comment (yvanhenang) [reply tsconfig.node.tsbuildinfo]: https://vite-pwa-org.netlify.app/ 
I've use this plugin

**Review by nancymuyeh — COMMENTED** [review]
@yvanhenang please make to sign your commits

**Comment by yvanhenang** [comment]
Hi @nancymuyeh , I have fixed the issue and now my commits are signed. Thank you for pointing it out!"

**Review by Koufan-De-King — CHANGES_REQUESTED** [review]
Hello @yvanhenang,

We are not yet at the stage where eslint is configured so you should remove the **eslint.config.js** file

Also, you are missing the following files
- LICENSE
- pwa-assets.config.ts and 
- README.md

Please create them and populate them with the appropriate content. Use [my pull request](https://github.com/ADORSYS-GIS/webank-UserApp/pull/4/files) as template

**Pull request closed without merging**